### PR TITLE
Fix urls. Remove unnecessary whitespace.

### PR DESCRIPTION
**Pull request by kmcphillips**
Some small cleanups. All whitespace changes unless mentioned inline. 😄

### CODE REVIEW

File: do-article-template.md.txt [hunk 1]
```diff
@@ -1,29 +1,28 @@
 <!--
-This is an article template you can use as a quick starting point when writing DigitalOcean tutorials. Once you've reviewed the template, delete the comments and begin writing your article. You'll find some examples of our custom Markdown at the very bottom of the template. 
+This is an article template you can use as a quick starting point when writing DigitalOcean tutorials. Once you've reviewed the template, delete the comments and begin writing your article. You'll find some examples of our custom Markdown at the very bottom of the template.
 
 As you write, refer to our style and formatting guidelines for more detailed explanations:
 
-- [do.co/style](do.co/style)
+- [do.co/style](https://do.co/style)
```
Comment (kmcphillips) [inline do-article-template.md.txt:6]: Less ambiguous to fill the whole URL out here.

File: do-article-template.md.txt [hunk 2]
```diff
@@ -1,29 +1,28 @@
...
 
 
-# How To [Install/Configure/Do Something] on [Distribution] 
+# How To [Install/Configure/Do Something] on [Distribution]
 
-<!--  Use Title Case for all Titles -->
+<!-- Use Title Case for all Titles -->
 
-<!-- Learn about the title, introduction, and Goals sections at https://do/co/style#title-introduction-and-goals  -->
+<!-- Learn about the title, introduction, and Goals sections at https://do.co/style#title-introduction-and-goals -->
```
Comment (kmcphillips) [inline do-article-template.md.txt:18]: This was linking to `do/co` which is wrong. Should be `do.co`

**Comment by kmcphillips** [comment]
Is this not maintained? @napcs ?

**Comment by napcs** [comment]
@kmcphillips hey :) It's totally maintained. I just never saw a notification on this for some reason. THanks for the ping.

**Comment by kmcphillips** [comment]
Thanks!

**Pull request merged**